test/dbverify.c

```diff
@@ -116,13 +116,29 @@ compare_databases(WacomDeviceDatabase *orig, WacomDeviceDatabase *new)
 	free(newall);
 }
 
+/**
+ * djb2 by Daniel J. Bernstein.
+ * http://www.cse.yorku.ca/~oz/hash.html
+ */
+unsigned long
+hash(const char *str)
+{
+	unsigned long hash = 5381;
+	int c;
+
+	while ((c = *str++))
+		hash = ((hash << 5) + hash) + c; /* hash * 33 + c */
+
+	return hash;
+}
+
 /* write out the current db, read it back in, compare */
 static void
 compare_written_database(WacomDeviceDatabase *db)
 {
 	char *dirname;
 	WacomDeviceDatabase *db_new;
-        WacomDevice **device, **devices;
+    WacomDevice **device, **devices;
```

Comment:
this looks like an indentation issue?

Reply:
This line was using spaces for indentation, but as I can see from the rest of the file, it should use tabs. I've updated the commit, now should be correct.